lib/teleterm/clusters/cluster_gateways.go

```diff
@@ -61,5 +63,29 @@ func (c *Cluster) CreateGateway(ctx context.Context, params CreateGatewayParams)
 		return nil, trace.Wrap(err)
 	}
 
+	cliCommand, err := buildCliCommand(c, gw)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	gw.CliCommand = cliCommand.String()
+
 	return gw, nil
 }
+
+func buildCliCommand(c *Cluster, gw *gateway.Gateway) (*exec.Cmd, error) {
+	routeToDb := tlsca.RouteToDatabase{
+		ServiceName: gw.TargetName,
+		Protocol:    gw.Protocol,
+		Username:    gw.TargetUser,
+	}
+
+	cmd, err := dbcmd.NewCmdBuilder(c.clusterClient, &c.status, &routeToDb, c.URI.GetRootClusterName(),
+		dbcmd.WithLocalProxy(gw.LocalAddress, gw.LocalPortInt(), ""),
```

Comment:
Not sure if it the last argument - caPath should be empty. For example mongo, and redis commands are using this arg to overwrite CA https://github.com/gravitational/teleport/blob/97c0b5cb835c6f55f2f1ba4347d4e40b6dfdbd53/tool/tsh/dbcmd.go#L333

Reply:
They do this only in the absence of the `noTLS` option and this callsite here will always use `dbcmd.WithNoTLS()`.

Perhaps in the future it'd be worthwhile to refactor `connectionCommandOpts` so that it's not possible to pass mutually exclusive options.